### PR TITLE
great new content

### PR DESCRIPTION
**Pull request by alixkendal**
Added a new p and img element

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,5 +9,8 @@
 <body>
     <h1>Git Pull Requests</h1>
     <h2>Let's see what happens...</h2>
+
+    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Dolorem deserunt, error, esse, obcaecati delectus possimus unde officiis vero odio quisquam mollitia quia magni? Nam, praesentium. Quos explicabo nisi ipsam hic.</p>
```
Comment (rachellehanna) [inline index.html:13]: This looks great!

**Review by rachellehanna — CHANGES_REQUESTED** [review]
Looks good! Maybe add another p under the img?

**Comment by alixkendal** [comment]
Made requested changes.

**Review by rachellehanna — APPROVED** [review]
Go ahead and merge!

**Pull request merged**